content/scripts/build.js

```diff
@@ -172,11 +173,10 @@ function runBuild(sources, options, logger) {
   } else if (options.ensureTitles) {
     return builder.ensureAllTitles();
   } else {
+    builder.initSelfHash();
+    builder.ensureAllTitles();
```

Comment:
Important! This way, the `this.allTitles` is always made available always indendent of starting the watcher or doing a full build from the CLI.